<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Control</class>
 <widget class="QWidget" name="Control">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>728</width>
    <height>728</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>728</width>
    <height>728</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>728</width>
    <height>728</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <property name="styleSheet">
   <string notr="true">*{
 border:none;
	
	
	
}
</string>
  </property>
  <widget class="QSlider" name="horizontalSlider">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>620</y>
     <width>641</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QSlider" name="volumeSlider">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>450</y>
     <width>20</width>
     <height>191</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="musicList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>660</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic2/1.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pre">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>660</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background-image: url(:/pic2/2.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="play">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>660</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic2/4.png);
  </string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="next">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>660</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic2/3.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="volume">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>660</y>
     <width>61</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic2/5.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>641</width>
     <height>561</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic2/IMG_20231001_164635.jpg);</string>
   </property>
  </widget>
  <widget class="QLabel" name="position">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>590</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="duration">
   <property name="geometry">
    <rect>
     <x>630</x>
     <y>590</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>00:00</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>731</width>
     <height>661</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic2/IMG_20231001_164635.jpg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>650</y>
     <width>731</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic2/20160628141711_ZxaJA.jpeg);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="playMode">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>660</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic2/7.png);
repeat: no-repeat;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="loveSong">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>660</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-image: url(:/pic2/14.png);
</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <zorder>label_2</zorder>
  <zorder>label</zorder>
  <zorder>horizontalSlider</zorder>
  <zorder>volumeSlider</zorder>
  <zorder>musicList</zorder>
  <zorder>pre</zorder>
  <zorder>play</zorder>
  <zorder>next</zorder>
  <zorder>volume</zorder>
  <zorder>listWidget</zorder>
  <zorder>position</zorder>
  <zorder>duration</zorder>
  <zorder>playMode</zorder>
  <zorder>loveSong</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
